### DISCUSSION TITLE
Graphql v1/v2 endpoints

### DISCUSSION THREAD

**Discussion by radoslavirha** [TypedProject/ts-express-decorators#2470] [Q&A]
Hi, I'm new in Ts.ED and I'm exploring this library. I was playing with GraphQL and I wanted to create two endpoints `/v1/graphql` and `/v2/graphql`. I don't need it currently, I'm just goofing around, but it doesn't work.

I forked template repo and created https://github.com/radoslavirha/tsed-example-graphql as an example.

My first question is if everything is set correctly? I think yes, but still.

I expect query
```
{
    recipes {
        id
        title
        description
    }
}
```
sent to `http://0.0.0.0:8083/v1/graphql` should return id 1 and `http://0.0.0.0:8083/v2/graphql` should return id 2.

It seems import order is important [here](https://github.com/radoslavirha/tsed-example-graphql/blob/54cb583162bf2b68892acf675b3354e7d0a0575d/src/Server.ts#L10)

Both endpoints returns result from first imported module, even when they are separated, have separate path and resolvers.

Thanks

**Comment by Romakita** [comment]
Ok it’s clearly a bug. Can open issue please?

**Reply by Romakita** [reply]
@radoslavirha can you open a bug please?

**Reply by radoslavirha** [reply]
Sorry I'm late.

**Reply by Romakita** [reply]
Fixed :)
